From 3d924bc9cb207a26d7b2f4b447d19c8e7f637822 Mon Sep 17 00:00:00 2001
From: StakeandRelax-Validator <stakeandrelax_validator@proton.me>
Date: Tue, 10 Feb 2026 20:45:03 +0000
Subject: [PATCH 4/4] build: derive version from genesis tag instead of cronos
 tag

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 5932c61..0cd78e2 100644
--- a/Makefile
+++ b/Makefile
@@ -105,10 +105,10 @@ endif
 
 all: build
 build: check-network print-ledger go.sum
-        @go build -mod=readonly $(BUILD_FLAGS) -o $(BUILDDIR)/$(BINARY_NAME) ./cmd/cronosd
+	@go build -mod=readonly $(BUILD_FLAGS) -o $(BUILDDIR)/$(BINARY_NAME) ./cmd/cronosd
 
 install: check-network print-ledger go.sum
-        @go build -mod=readonly $(BUILD_FLAGS)  -o $${GOBIN:-$$(go env GOPATH)/bin}/$(BINARY_NAME) ./cmd/cronosd
+	@go build -mod=readonly $(BUILD_FLAGS)  -o $${GOBIN:-$$(go env GOPATH)/bin}/$(BINARY_NAME) ./cmd/cronosd
 
 test: test-memiavl test-store
 	@go test -tags=objstore -v -mod=readonly $(PACKAGES) -coverprofile=$(COVERAGE) -covermode=atomic
-- 
2.43.0

